lib/cell.rb

```diff
@@ -0,0 +1,6 @@
+class Cell
```

Comment:
I think it's redundant to create a Cell class.. The Board class should be enough to handle all board related logic.